[
  {
    "type": "Error",
    "date": "2025-03-06 / 07:42",
    "errorLocation": "    at Timeout._onTimeout (/home/losss/web/wabot.websiteku.help/public_html/minebot/node_modules/minecraft-protocol/src/client/keepalive.js:18:28)",
    "errorMessage": "client timed out after 30000 milliseconds",
    "error": "client timed out after 30000 milliseconds\nError: client timed out after 30000 milliseconds\n    at Timeout._onTimeout (/home/losss/web/wabot.websiteku.help/public_html/minebot/node_modules/minecraft-protocol/src/client/keepalive.js:18:28)\n    at listOnTimeout (node:internal/timers:573:17)\n    at process.processTimers (node:internal/timers:514:7)"
  },
  {
    "type": "Error",
    "date": "2025-03-06 / 07:47",
    "errorLocation": "    at TCP.onStreamRead (node:internal/stream_base_commons:218:20)",
    "errorMessage": "read ECONNRESET",
    "error": "read ECONNRESET\nError: read ECONNRESET\n    at TCP.onStreamRead (node:internal/stream_base_commons:218:20)"
  },
  {
    "type": "Error",
    "date": "2025-03-06 / 07:56",
    "errorLocation": "    at TCP.onStreamRead (node:internal/stream_base_commons:218:20)",
    "errorMessage": "read ECONNRESET",
    "error": "read ECONNRESET\nError: read ECONNRESET\n    at TCP.onStreamRead (node:internal/stream_base_commons:218:20)"
  },
  {
    "type": "Error",
    "date": "2025-03-11 / 19:36",
    "errorLocation": "    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)",
    "errorMessage": "getaddrinfo ENOTFOUND xetapixel.com",
    "error": "getaddrinfo ENOTFOUND xetapixel.com\nError: getaddrinfo ENOTFOUND xetapixel.com\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"
  },
  {
    "type": "Error",
    "date": "2025-03-12 / 00:25",
    "errorLocation": "    at Timeout._onTimeout (/home/losss/web/wabot.websiteku.help/public_html/minebot/node_modules/minecraft-protocol/src/client/keepalive.js:18:28)",
    "errorMessage": "client timed out after 30000 milliseconds",
    "error": "client timed out after 30000 milliseconds\nError: client timed out after 30000 milliseconds\n    at Timeout._onTimeout (/home/losss/web/wabot.websiteku.help/public_html/minebot/node_modules/minecraft-protocol/src/client/keepalive.js:18:28)\n    at listOnTimeout (node:internal/timers:573:17)\n    at process.processTimers (node:internal/timers:514:7)"
  },
  {
    "type": "Error",
    "date": "2025-03-12 / 05:01",
    "errorLocation": "    at Timeout._onTimeout (/home/losss/web/wabot.websiteku.help/public_html/minebot/node_modules/minecraft-protocol/src/client/keepalive.js:18:28)",
    "errorMessage": "client timed out after 30000 milliseconds",
    "error": "client timed out after 30000 milliseconds\nError: client timed out after 30000 milliseconds\n    at Timeout._onTimeout (/home/losss/web/wabot.websiteku.help/public_html/minebot/node_modules/minecraft-protocol/src/client/keepalive.js:18:28)\n    at listOnTimeout (node:internal/timers:573:17)\n    at process.processTimers (node:internal/timers:514:7)"
  },
  {
    "type": "Error",
    "date": "2025-03-12 / 08:34",
    "errorLocation": "    at Timeout._onTimeout (/home/losss/web/wabot.websiteku.help/public_html/minebot/node_modules/minecraft-protocol/src/client/keepalive.js:18:28)",
    "errorMessage": "client timed out after 30000 milliseconds",
    "error": "client timed out after 30000 milliseconds\nError: client timed out after 30000 milliseconds\n    at Timeout._onTimeout (/home/losss/web/wabot.websiteku.help/public_html/minebot/node_modules/minecraft-protocol/src/client/keepalive.js:18:28)\n    at listOnTimeout (node:internal/timers:573:17)\n    at process.processTimers (node:internal/timers:514:7)"
  },
  {
    "type": "Error",
    "date": "2025-03-14 / 15:47",
    "errorLocation": "    at Timeout._onTimeout (/home/losss/web/wabot.websiteku.help/public_html/minebot/node_modules/minecraft-protocol/src/client/keepalive.js:18:28)",
    "errorMessage": "client timed out after 30000 milliseconds",
    "error": "client timed out after 30000 milliseconds\nError: client timed out after 30000 milliseconds\n    at Timeout._onTimeout (/home/losss/web/wabot.websiteku.help/public_html/minebot/node_modules/minecraft-protocol/src/client/keepalive.js:18:28)\n    at listOnTimeout (node:internal/timers:573:17)\n    at process.processTimers (node:internal/timers:514:7)"
  },
  {
    "type": "Error",
    "date": "2025-03-16 / 00:00",
    "errorLocation": "    at Timeout._onTimeout (/home/losss/web/wabot.websiteku.help/public_html/minebot/node_modules/minecraft-protocol/src/client/keepalive.js:18:28)",
    "errorMessage": "client timed out after 30000 milliseconds",
    "error": "client timed out after 30000 milliseconds\nError: client timed out after 30000 milliseconds\n    at Timeout._onTimeout (/home/losss/web/wabot.websiteku.help/public_html/minebot/node_modules/minecraft-protocol/src/client/keepalive.js:18:28)\n    at listOnTimeout (node:internal/timers:573:17)\n    at process.processTimers (node:internal/timers:514:7)"
  },
  {
    "type": "Error",
    "date": "2025-03-16 / 00:03",
    "errorLocation": "    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1606:16)",
    "errorMessage": "connect ETIMEDOUT 185.207.164.1:25565",
    "error": "connect ETIMEDOUT 185.207.164.1:25565\nError: connect ETIMEDOUT 185.207.164.1:25565\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1606:16)"
  },
  {
    "type": "Error",
    "date": "2025-03-16 / 00:05",
    "errorLocation": "    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1606:16)",
    "errorMessage": "connect ETIMEDOUT 185.207.164.1:25565",
    "error": "connect ETIMEDOUT 185.207.164.1:25565\nError: connect ETIMEDOUT 185.207.164.1:25565\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1606:16)"
  },
  {
    "type": "Error",
    "date": "2025-03-16 / 00:08",
    "errorLocation": "    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1606:16)",
    "errorMessage": "connect ETIMEDOUT 185.207.164.1:25565",
    "error": "connect ETIMEDOUT 185.207.164.1:25565\nError: connect ETIMEDOUT 185.207.164.1:25565\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1606:16)"
  },
  {
    "type": "Error",
    "date": "2025-03-16 / 00:10",
    "errorLocation": "    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1606:16)",
    "errorMessage": "connect ETIMEDOUT 185.207.164.1:25565",
    "error": "connect ETIMEDOUT 185.207.164.1:25565\nError: connect ETIMEDOUT 185.207.164.1:25565\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1606:16)"
  }
]